### PR TITLE
date field added to cohorts table

### PR DESCRIPTION
**Pull request by AamirahV**
Problem: Needed a date column inside the Cohorts table

Solution: Added a date column to Cohorts table using a migration file and sequelize

Jira issue: https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?selectedIssue=AT-3

### CODE REVIEW

File: migrations/20210817094012-create-added-date-field-to-cohorts.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+     await queryInterface.addColumn(
```
Comment (dearshrewdwit) [inline migrations/20210817094012-create-added-date-field-to-cohorts.js:5]: Is there a reason for the args to be on separate lines?

**Review by dearshrewdwit — COMMENTED** [review]
Nice! Thanks for the small PR.

Quick question - see below

File: migrations/20210817094012-create-added-date-field-to-cohorts.js (hunk 1 above)
Comment (breeshidy) [reply migrations/20210817094012-create-added-date-field-to-cohorts.js:5]: No, just thought it was easier to read

**Review by dearshrewdwit — DISMISSED** [review]

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Nice. Good idea.

Remember - the project doesn't have a date column field yet, so you don't need to add the date column, remove it, and then add the start date column.

Simply add the startDate column!

**Comment by AamirahV** [comment]
> Nice. Good idea.
> 
> Remember - the project doesn't have a date column field yet, so you don't need to add the date column, remove it, and then add the start date column.
> 
> Simply add the startDate column!

Done

**Comment by breeshidy** [comment]
> @AamirahV let's get this merged. You need 1 more review

can i approve this @dearshrewdwit

**Comment by dearshrewdwit** [comment]
2 reviews are needed - 1 internal and 1 external.

Internal - to your team (but not one of the PR authors)
External - any non-team member

**Review by grantwhiteman — APPROVED** [review]
Looks good

**Review by dearshrewdwit — APPROVED** [review]

**Pull request merged**